### DISCUSSION TITLE
Account Locked Out By 2FA

### DISCUSSION THREAD

**Discussion by zhangshuo1231** [community/community#144543] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### GitHub Feature Area

Actions

### Body

I have another GitHub account which I can't login anymore. That account had 2FA configured months ago. Last week, when I tried to login, GitHub blocked me after I typed in the code. Wondering if my TOTP app had some issue, I unfortunately uninstalled and reinstalled the TOTP on my mobile. Then I realized that I couldn't add back the GitHub account to my TOTP anymore. Because in order to configure the TOTP, I need to login and get the QR code, but in order to login, I need the TOTP to work. It's a deadlock.

Any ideas? Thank you.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

Thank you for your patience. The community is unable to assist with nor escalate with this account-related situation. Please review our doc: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). Within the doc, you also have the option to go through recovery with a Virtual Agent. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
